app/models/feed.rb

```diff
@@ -161,6 +161,6 @@ def saved_search_belongs_to_feed_teams
   end
 
   def create_feed_team
-    self.teams << self.team unless self.team.nil?
+    FeedTeam.create!(feed: self, team: self.team, shared: true) unless self.team.nil?
```

Comment:
Due to current product requirements, all new feeds should be `shared` for now.

Reply:
This is the same as sending the mutation with `published: true`, right?

Reply:
No, `published` is a property of the `Feed`, while `shared` is a property of the `FeedTeam`.

Reply:
:exploding_head: